packages/react/src/components/Combobox/Combobox.tsx

```diff
@@ -439,11 +650,46 @@ const Combobox = forwardRef<HTMLDivElement, ComboboxProps>(
           // there's already keyboard handlers to open the listbox on the input element
           onClick={handleInputClick}
         >
+          {multiselect &&
+            selectedValues.map((value, index) => {
+              const refCallback = (elem: HTMLButtonElement | null) => {
+                if (elem) {
+                  pillsRef.current[index] = elem;
+                } else {
+                  pillsRef.current.splice(index, 1);
+                }
+              };
+
+              const handleClick = () =>
+                handleRemove(pillsRef.current[index], value);
+
+              const commonProps = {
+                ref: refCallback,
+                key: value,
+                className: 'Combobox__pill',
+                tabIndex: -1,
+                onClick: handleClick,
+                onKeyDown: handlePillKeyDown
+              };
+
+              return !disabled ? (
+                <TagButton
+                  label=""
+                  value={value || ''}
+                  icon="close"
+                  {...commonProps}
+                />
+              ) : (
+                <Button disabled={disabled} {...commonProps}>
+                  {value}
+                </Button>
```

Comment:
These 2 components do not have similar styles. You can hold off on changes here until I confirm with @dequelabs/cauldron-design how we want these pills to appear.

Reply:
It is not possible to have TagButton without icon, that's why I'we added a conditional here to render button instead

Reply:
Added a screenshot on how it would look like